test/e2e-go/cli/goal/cicd_expect/goalAlgonetStatusTest.exp

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/expect -f
+set err 0
+log_user 1
+
+if { [catch {
+
+    set ::GLOBAL_TEST_ALGO_DIR ""
+    set ::GLOBAL_TEST_ROOT_DIR ""
+    set ::GLOBAL_NETWORK_NAME ""
+
+    source  ../expect/goalExpectCommon.exp
+    set TEST_ALGO_DIR [lindex $argv 0]
+    set TEST_DATA_DIR [lindex $argv 1]
+
+    puts "TEST_ALGO_DIR: $TEST_ALGO_DIR"
+    puts "TEST_DATA_DIR: $TEST_DATA_DIR"
+
+    set TIME_STAMP [clock seconds]
+
+    set TEST_ROOT_DIR $TEST_ALGO_DIR/data
+
+    set NODE_NAME [exec ls $TEST_ALGO_DIR/data]
+    set TEST_PRIMARY_NODE_DIR $TEST_ROOT_DIR/$NODE_NAME
+
+    exec sleep 5
```

Comment:
I'm not opposing to sleep for 5 seconds.. but why do we need to ?

Reply:
This is required to allow the nodes to initialize.